From 056083c3a9b495f380ff4c7c8cd13aeca908a066 Mon Sep 17 00:00:00 2001
From: Stella Laurenzo <stellaraccident@gmail.com>
Date: Tue, 29 Apr 2025 22:13:17 -0700
Subject: [PATCH 5/9] Adapt to some upstream hipblas-common usage differences.

* The hipblas-common library seems to be in the process of being renamed to hip::hipblas-common but not uniformly yet.
* This was coming from a dependency in a prior state, thus INTERFACE was ok. However, it is needed within the library and with the dependent not advertising it anymore, compilation fails to find hipblas-common.h.
---
 library/src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/src/CMakeLists.txt b/library/src/CMakeLists.txt
index 5da9875..0b3b6f1 100755
--- a/library/src/CMakeLists.txt
+++ b/library/src/CMakeLists.txt
@@ -71,7 +71,7 @@ add_library( roc::hipblas ALIAS hipblas )
 set(static_depends)
 
 find_package( hipblas-common REQUIRED CONFIG PATHS ${ROCM_PATH})
-target_link_libraries( hipblas INTERFACE roc::hipblas-common )
+target_link_libraries( hipblas PUBLIC hip::hipblas-common )
 
 # Build hipblas from source on AMD platform
 if(HIP_PLATFORM STREQUAL amd)
-- 
2.47.1.windows.2

